app/javascript/tests/mockData/mockData.js

```diff
@@ -0,0 +1,22 @@
+import mockAnalyticsData from './mockAnalytics'
+
+const mockUser1 = {
```

Comment:
I think we will probably want a `mockUsers.js` file in `mockData/`

Reply:
done